src/app/home/pages/PlansPage/helpers.ts

```diff
@@ -94,7 +94,10 @@ export const getPlanInfo = (plan: IPlan) => {
 };
 
 export const getMigrationTypeFromPlan = (plan: IMigPlan): MigrationType => {
-  let migrationType = plan?.metadata?.annotations['migration.openshift.io/selected-migplan-type'];
+  let migrationType: MigrationType;
+  if (plan?.metadata.annotations) {
```

Comment:
```suggestion
  if (plan.metadata?.annotations) {
```

I think with the optional chaining, all of the values before the first '?' are implicitly checked for undefined. I am guilty of doing this to, but its something we can clean up along the way.